server/members/method.js

```diff
@@ -0,0 +1,17 @@
+Meteor.methods({
```

Comment:
This method could be moved to either SG/methods.js or users/methods.js

Reply:
move to SG/methods.js